agents/human_agent.py

```diff
@@ -8,8 +9,31 @@ class HumanAgent(Agent):
     the user choose the action.
     TODO - implement everything!
     """
+
     def __init__(self, name):
         super(HumanAgent, self).__init__(name)
 
     def choose_action(self, env):
-        pass
\ No newline at end of file
+        # create list of available moves
+        available_moves = env.available_moves(self)
+
+        # check if the list are not empty
+        if available_moves:
+
+            # action of the player
+            while True:
+                print("Please choose your action from the list below and press ENTER:")
+                for act in available_moves:
+                    print(act)
```

Comment:
Don't make the user type out the action as we see it... in the game `4-in-row` the move may be represented by a single integer but in another game it could be represented by a tuple of ints, or even by a floating point number.  
Instead - please make a numbered list of actions, something in the format of:
```
1= act1
2= act2
3= act3
```
And the player only has to type the serial number of the choice, e.g. if he chooses `act2` he only has to type the number `2`.